pkg/devconsole/proxy/types.go

```diff
@@ -0,0 +1,15 @@
+package proxy
+
+type RequestBody struct {
+	Method      string              `json:"method"`
+	Url         string              `json:"url"`
+	Headers     map[string][]string `json:"headers"`
+	Queryparams map[string][]string `json:"queryparams"`
+	Body        string              `json:"body"`
+}
+
+type ResponseBody struct {
```

Comment:
Your request "body" has a field `Body String`. That's a good indicator that this type isn't the full body:

```suggestion
type ProxyResponse struct {
```